rpc_jobs/rpc_octavia.yml

```diff
@@ -10,6 +10,8 @@
     # upstream installer
     branches:
       - "master"
+      - "Pike"
```

Comment:
Please use lower case for branch names. I see this is the name in rpc-octavia too, so that should be renamed. Being consistent with this reduces confusion and makes it easier for us to build validation into tests.